README.md

```diff
@@ -51,7 +55,7 @@ The "bar" for including code in this library is higher than that seen in our jup
 
 ### Code Style
 
-This repo follows the Osmo [Code Style Manifesto](https://docs.google.com/document/d/1W1Ipug8IACL4PfZAq5bQKlmfcJGmHGKNH95_FwJcjaI).
+This repo follows the Osmo [Code Style Manifesto](https://docs.google.com/document/d/1W1Ipug8IACL4PfZAq5bQKlmfcJGmHGKNH95_FwJcjaI) as applies to Python code.
```

Comment:
Perhaps a hashed link that jumps to the Python section?  I can see the value of needing to look at the overview before getting there.

Reply:
there isn't really just one python section that covers everything. a bunch of the general stuff applies too.